store/postgres/src/relational/ddl.rs

```diff
@@ -116,19 +116,26 @@ impl Table {
             Ok(cols)
         }
 
+        let vid_type = if !self.object.is_object_type() {
```

Comment:
This would be easier to read if it was not negated. And maybe has a comment that the `vid` of aggregations are not necessarily in insertion order